Composer/packages/client/src/components/ManageService/ManageService.tsx

```diff
@@ -210,9 +219,21 @@ export const ManageService = (props: ManageServiceProps) => {
         setShowAuthDialog(true);
       }
       newtoken = getTokenFromCache('accessToken');
-    }
-    if (newtoken) {
-      setToken(newtoken);
+      if (newtoken) {
+        const decoded = decodeToken(newtoken);
+        if (decoded) {
+          setToken(newtoken);
+          setUserProvidedTokens(true);
+        } else {
+          setTenantsErrorMessage(
+            formatMessage(
```

Comment:
Does this happen only in dev mode? or a normal user might face this message as well? If so, is this UX approved?
I think we need to provide a much simpler way to clear the token from application local storage, as simple as having a link/button right here to clear it.

Reply:
This only happens in dev mode.